src/constants.ts

```diff
@@ -24,3 +24,12 @@ export const revisions = {
     validNotFound: validRevisionsNotFound,
     invalid: invalidRevisions,
 }
+
+const faucetMnemonic =
```

Comment:
wonder if we should grab it from github secrets or it is ok to keep it there ?

Reply:
Nope, no need, we are using the public mnemonic that is printed to the console when you do `thor solo`